pubspec.yaml

```diff
@@ -4,16 +4,16 @@ description: |
   a plugin package for dart and flutter apps.
   Developers can build ones to interact with Dfinity's blockchain directly.
 repository: https://github.com/AstroxNetwork/agent_dart
-version: 1.0.0-dev.14
+version: 1.0.0-dev.15
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
-  flutter: '>=3.0.0'
+  sdk: '>=2.17.0 <4.0.0'
+  flutter: '>=3.0.0 <4.0.0'
 
 dependencies:
   archive: ^3.3.0
   args: ^2.3.1
-  basic_utils: ^4.2.2
+  basic_utils: ^5.5.4
```

Comment:
If those dependencies require Dart >= 3.0, we should update our sdk constriant too.